src/main/java/org/parabot/core/desc/ServerProviderInfo.java

```diff
@@ -50,7 +50,16 @@ public ServerProviderInfo(URL providerInfo, String username, String password) {
         }
     }
 
-    public ServerProviderInfo(String clientJar, String hooks, String name, String clientClass, int bankTabs) {
+    /**
+     * Initialize configuration with data provided by {@link org.parabot.core.parsers.servers.LocalServers} from a {@code /parabot/servers/config.json} file. Also loads the default Settings map from the BDN.
+     * @param clientJar Name of the client jar file
+     * @param hooks Name of the hooks file
+     * @param name Server name
+     * @param clientClass Entry class within the client jar
+     * @param bankTabs Bank tabs - only relevant for certain servers. Default 0
+     * @param randoms
+     */
+    public ServerProviderInfo(String clientJar, String hooks, String name, String clientClass, int bankTabs, String randoms) {
```

Comment:
This is not backward compatible. Could you instead create a "new" constructor with the old parameters that basically calls this constructor?